Sources/SwiftKafka/KafkaConsumer.swift

```diff
@@ -183,7 +223,7 @@ public final class KafkaConsumer: Sendable, Service {
                     throw error
                 }
                 try await Task.sleep(for: self.config.pollInterval)
-            case .pollUntilClosed(let client):
+            case .pollWithoutYield(let client):
```

Comment:
This also means that we are polling the queue and all messages will just get dropped and never committed. I think this is the sane thing to do. We should add this as a comment here.